installer_linux/make_deb.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# preflight check
+
+if [[ ! -f ./make_deb.sh ]]; then
+	echo "You must run this script from within the installer_osx directory!"
+	exit 1
+fi
+
+# version
+
+if [ "$SURGE_VERSION" != "" ]; then
+    VERSION="$SURGE_VERSION"
+elif [ "$1" != "" ]; then
+    VERSION="$1"
+fi
+
+if [ "$VERSION" == "" ]; then
+    echo "You must specify the version you are packaging!"
+    echo "eg: ./make_deb.sh 1.0.6b4"
+    exit 1
+fi
+
+# Valid version for DEB start with a digit.
+DEB_VERSION=$VERSION
+if [[ ${DEB_VERSION:0:1} =~ [0-9] ]]; then
+    echo "DEB VERSION same as Version"
+else
+    DEB_VERSION="0.${VERSION}"
+    echo "DEB VERSION is ${DEB_VERSION}"
+fi
+
+
+# Names
+SURGE_NAME=Surge
+# SURGE_NAME=surge-synthesizer
+
+# locations
+
```

Comment:
maybe `rm -rf surge-synthesizer` to make sure a failed prior run doesn't leave bad stuff around?